### ISSUE TITLE
Post new planet.m.o in room

### ISSUE THREAD

**Issue by dannycolin** [bnjbvr/botzilla#2]
Would be nice to have the same function as firebot on IRC that automatically post new articles from planet.m.o

**Comment by bnjbvr** [comment]
Nice, I didn't know it could do this! Well, I think that an RSS check in setInterval could work to retrieve the titles. Now the bot only knows how to react to messages, and can't just tell things from nowhere.

Also, I think Matrix already has support for some native integrations, which include RSS feed reading.

A serious question is whether this would address the safety requirements of the Mozilla CPG; in particular, if something was offensive in a title of a blog post published via planet.m.o, then showing it in the channel could be offensive. I assume there's some after-the-fact editorial control on Planet, but this wouldn't be sufficient since a Matrix message could be posted before after-the-fact edition happened.

**Comment by bnjbvr** [comment]
Actually, this would require the (already existing) RSS bot to live in the same instance, since Mozilla's instance isn't federated.

**Comment by dannycolin** [comment]
> A serious question is whether this would address the safety requirements of the Mozilla CPG; in particular, if something was offensive in a title of a blog post published via planet.m.o

Good point. I haven't thought about the fact that it isn't moderated before appearing on planet.m.o. Maybe instead of using planet.m.o's RSS we could have an RSS feed of all the Mozilla Blogs.

**Comment by auscompgeek** [comment]
If you don't feel like reinventing the wheel too much, the RSS bot listed in the Riot integration manager is [go-neb](https://github.com/matrix-org/go-neb).

The [matrix.org bots page](https://matrix.org/docs/projects/bots/) also lists another RSS bot.
